### ISSUE TITLE
Header completion list does not trigger after previous completion

### ISSUE THREAD

**Issue by kiwixz** [clangd/clangd#395]
First of all, thanks for implementing #366 it will save me a lot of time.

It seems the list displays when typing some characters ie `</`, but accepting a completion will enter a folder including the terminating slash and wont trigger list for next element in path.

Hitting backspace then slash does show the list indeed.

On a related note, vscode suggest `#include <header>` when typing `#i` but it suffers from the same problem.

**Comment by kadircet** [comment]
tl;dr; unfortunately clangd is just a language server and can only report completions when clients request it. You can force-request a completion through ctrl+space in vscode.

it doesn't make much sense for clients to trigger code completion again immediately after user accepts a suggestion though. so i suppose we should drop the trailing slash instead and let the user type it so that clients can send a code completion request again.

i remember a discussion around including closing `"` and `>` in the `textEdit` but not so sure about `/`. if there wasn't a deliberate reason for including that in the `textEdit` i think we should just drop that. (unless LSP starts providing servers with a way to "push" completions or somehow signal clients that they should request completion again).

**Comment by puremourning** [comment]
> i remember a discussion around including closing " and > in the textEdit but not so sure about /

IMO don’t include the / for the same reason as the > and “

**Comment by DavidGoldman** [comment]
The VS Code extension could improve upon this by adding the following the package.json (as well as for other languages):

```
  "configurationDefaults": {
      "[cpp]": {
        "editor.quickSuggestions": {
          "strings": true
        }
      }
    }
```

although it might have other downsides (e.g. triggering code-completion and/or snippets in regular non-import strings) that would need to be handled.

We could also remove the `/` in the completion and then let typing the `/` automatically trigger completion of the next item.
